app/src/main/java/de/ur/explure/viewmodel/BottomNavViewModel.kt

```diff
@@ -11,26 +10,10 @@ class BottomNavViewModel : ViewModel(), KoinComponent {
 
     private val mainAppRouter: MainAppRouter by inject()
```

Comment:
Wieso konnten wir hier den Router nicht einfach über den Konstruktor injecten?

Reply:
Ist zwar jetzt geändert, aber fürs Verständnis:
Zwischen Konstruktor und inject() ist kein sonderlich großer Unterschied. Der Konstruktor wird halt schon beim Indexen überprüft und dadurch die Depenedencies überprüft. Und ich glaube Konstruktor ist bisschen effizienter. Hier was das Problem, dass ich das ViewModel als activityViewModel in ein Fragment injecten wollte. Das ViewModel hätte so den lifecycle der Activity gehabt. Dafür benötigt man aber einen leeren Kosntruktor.